### DISCUSSION TITLE
pls help code linting

### DISCUSSION THREAD

**Discussion by bailbmogu** [cpeditor/cpeditor#1330] [Q&A]
CP editor is kind of messed up.
I followed everything this https://cpeditor.org/docs/setup/ said for code linting, yet my syntax errors still dont show up.
I will admit CP is very helpful but this is a pain.
pls help

**Comment by ouuan** [comment]
Please give more background information. How did you set it up?

**Comment by DodgeTheDarins** [comment]
rip
